iron-input.html

```diff
@@ -28,6 +28,15 @@
 
     <input is="iron-input" bind-value="{{myValue}}">
 
+`iron-input` of type "number" or "date" can use the `bind-value-as-number` property for two-way data binding.
```

Comment:
If you have this:
```html
<input is="iron-input" type="date" bind-value-as-number="3">
```
it will be the number of milliseconds from midnight UTC the morning of 1970-01-01
http://www.w3.org/TR/2011/WD-html5-20110405/common-input-element-attributes.html#dom-input-valueasnumber

Here how it works with a raw input (test it on chrome) 
http://jsbin.com/qebefogedi/edit?html,output